src/NServiceBus.NHibernate/Outbox/NSBDateTimeType.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace NServiceBus.Deduplication.NHibernate.Config
+﻿namespace NServiceBus.Outbox.NHibernate
```

Comment:
Is there any migration concern around this namespace changing since this type is referenced in `OutboxRecordMapping`? It's not clear to be what `OutboxRecordMapping` is doing, so I wanted to make sure the type's namespace wouldn't be baked into some database table or something like that.

Reply:
Good catch, I think we are fine but reverted the namespace just to be on the safe side